### PR TITLE
Fixing improper indendation of Enumerator and Repeaters 

### LINKED ISSUE #4116: Visual indicator of Enumerator and Repeaters
**Is your feature request related to a problem? Please describe.**
As a CiviForm Admin, I'd like a clear visual indicator of the screens that contain enumerators and those that are repeated screens with repeater questions. When building or reviewing a program, there is no clear way to see if a screen is an enumerator or repeated screen. 

**Describe the solution you'd like**
A visual clue to see the enumerator and repeated screens--could be indentation of the screens or other visual clue.

**Additional context**
image

### PR DESCRIPTION
**Pull request by swatkat1**
### Description

After the last Tailwind update, the visual indicators of Enums and Repeaters in the Program Block list went missing. This PR address the bug.  

## Release notes

After the last Tailwind update, the visual indicators of Enums and Repeaters in the Program Block list went missing. This PR address the bug.  

#### General

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release >
- [x] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [ ] Manually tested at 200% size
- [ ] Manually evaluated tab order

#### New Features

- [ ] Add new FeatureFlag env vars to `server/conf/application.conf`
- [ ] Conditioned new functionality on a [FeatureFlag](https://docs.civiform.us/contributor-guide/developer-guide/feature-flags)
- [ ] Wrote browser tests with the feature flag off and on, etc.

### Issue(s) this completes

Fixes #4116

### CODE REVIEW

File: server/app/views/admin/programs/ProgramBlockEditView.java [hunk 1]
```diff
@@ -263,7 +263,7 @@ private DivTag renderBlockList(
                   "flex-row",
                   "gap-2",
                   "py-2",
-                  "px-4",
+                  "px-" + (4 + (level * 2)),
```
Comment (gwendolyngoetz) [inline server/app/views/admin/programs/ProgramBlockEditView.java:266]: (blocking) It's hard to quickly understand what's going on. Pull the calculation up to it's own variable and assign the magic numbers to some constants.

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: server/app/views/admin/programs/ProgramBlockEditView.java (hunk 1 above)
Comment (swatkat1) [reply server/app/views/admin/programs/ProgramBlockEditView.java]: Done!

File: server/app/views/admin/programs/ProgramBlockEditView.java [hunk 2]
```diff
@@ -246,15 +246,16 @@ private DivTag renderBlockList(
...
     String genericBlockDivId = "block_list_item_";
     for (BlockDefinition blockDefinition : blockDefinitions) {
 
       // TODO: Not i18n safe.
       int numQuestions = blockDefinition.getQuestionCount();
       String questionCountText = String.format("Question count: %d", numQuestions);
       String blockName = blockDefinition.name();
-
+      // indendation value for enums and repeaters
+      int listIndendationFactor = 4 + (level * 2);
```
Comment (gwendolyngoetz) [inline server/app/views/admin/programs/ProgramBlockEditView.java:258]: (blocking) I'd still like to see the 4 and the 2 put in named constants. I don't fully understand what each value is. 

Something like BASE_INDENTATION_SIZE = 4. I'm not sure I understand what the 2 should be called.

**Review by gwendolyngoetz — CHANGES_REQUESTED** [review]

File: server/app/views/admin/programs/ProgramBlockEditView.java (hunk 2 above)
Comment (swatkat1) [reply server/app/views/admin/programs/ProgramBlockEditView.java:258]: TBH- the previous code calculated like this-   DivTag container = div().withClass("pl-" + level * 2); but the pl value was overridden after the tailwind update. I carried over similar calculation, but for px. may be call it "FACTOR_INCREASE_ON_LEVEL"?

**Review by gwendolyngoetz — APPROVED** [review]

**Pull request merged**